spec/ios_bump_version_release_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+
+describe Fastlane::Actions::IosBumpVersionReleaseAction do
+  let(:default_branch) { 'my_new_branch' }
+  let(:ensure_git_action_instance) { double() }
+  let(:versions) { ['6.30'] }
+  let(:next_version) { '6.31.0.0' }
+  let(:next_version_short) { '6.31' }
+
+  describe 'creates the release branch, bumps the app version and commits the changes' do
+    before do
+      allow(Fastlane::Action).to receive(:other_action).and_return(ensure_git_action_instance)
+      allow(ensure_git_action_instance).to receive(:ensure_git_branch).with(branch: default_branch)
```

Comment:
Nit: given how `other_action` worked in fastlane internally—i.e. it's a sort of _proxy object_, to call any other built-in Fastlane action from a fastlane plugin's implementation—I think it'd make more sense to call the mock just "`other_action`" or `other_action_mock` or something, rather than name the object returned by the `other_action` function call to be related to what method will be called on it later.

```suggestion
      other_action = double()
      allow(Fastlane::Action).to receive(:other_action).and_return(other_action)
      allow(other_action).to receive(:ensure_git_branch).with(branch: default_branch)
```

Reply:
Good point 👍